cspell.json

```diff
@@ -4,5 +4,8 @@
     "dictionaryDefinitions": [],
     "dictionaries": [],
     "ignoreWords": [],
-    "import": [".cspell-docs.json", ".cspell-code.json"]
+    "import": [
+        ".cspell-docs.json",
+        ".cspell-code.json"
+    ]
```

Comment:
tomato tomahto. Can probably just revert this change.
https://github.com/bacalhau-project/bacalhau/pull/3372/files#diff-77519f787fe7e051a14c588f101d61e4d8f0d3b01bf8e16cb65d3838243d4e68L8